File: modules/core/src/main/java/com/fapiko/jetbrains/plugins/better_direnv/settings/DirenvSettings.java
package com.fapiko.jetbrains.plugins.better_direnv.settings;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class DirenvSettings {
    private final boolean direnvEnabled;
    private final boolean direnvTrusted;
}

File: modules/core/src/main/kotlin/com/fapiko/jetbrains/plugins/better_direnv/settings/DirenvSettings.kt
package com.fapiko.jetbrains.plugins.better_direnv.settings

data class DirenvSettings(
    @JvmField val direnvEnabled: Boolean,
    @JvmField val direnvTrusted: Boolean
) {
    fun isDirenvEnabled(): Boolean = direnvEnabled
    fun isDirenvTrusted(): Boolean = direnvTrusted
}